angelleye-includes/angelleye-session-functions.php

```diff
@@ -61,6 +73,19 @@ function angelleye_unset_session($key) {
         }
 
     } catch (Exception $ex) {
+        
+    }
+}
 
+if (!function_exists('angelleye_session_init')) {
```

Comment:
@kcppdevelopers From now on for every function please include proper comments, for example. what it does and what will be the return value etc.

Reply:
okay sure